dask_cuda/benchmarks/local_cupy_transpose_sum.py

```diff
@@ -23,6 +45,10 @@
     ) as cluster:
         async with Client(cluster, asynchronous=True) as client:
 
+            if args.rmm:
+                await client.run(cudf.set_allocator, "default", pool=True)
+                await client.run(cupy.cuda.set_allocator, rmm_mem_allocator)
```

Comment:
What happens here if `args.rmm == True` but you couldn't import cudf or rmm? I think this will raise an error, no?

Reply:
Yes :)
Do you want me to raise a descriptive exception?

Reply:
I think that would be nice, either that or stating in the output that RMM wasn't imported and thus, skipped. Otherwise, the first persons who tries that and it fails will report this as a bug. :)